### ISSUE TITLE
hdays.py skips certain holidays for specific years?

### ISSUE THREAD

**Issue by kemo993** [facebook/prophet#1463]
Hello,

I've just noticed that `hdays.py` skips certain holidays for specific years, e.g. "Feast of the Sacrifice" holiday in 2020.

Consider the following example and corresponding output:
```python
fbprophet.make_holidays.make_holidays_df(year_list=[2019, 2020], country='TU')
```
```
           ds                                         holiday
0  2019-01-01                                  New Year's Day
1  2019-04-23       National Sovereignty and Children's Day\t
2  2019-05-01                                       Labor Day
3  2019-05-19  Commemoration of Atatürk, Youth and Sports Day
4  2019-07-15                Democracy and National Unity Day
5  2019-08-30                                     Victory Day
6  2019-10-29                                    Republic Day
7  2019-06-04                                     Eid al-Fitr
8  2019-08-12                          Feast of the Sacrifice
9  2020-01-01                                  New Year's Day
10 2020-04-23       National Sovereignty and Children's Day\t
11 2020-05-01                                       Labor Day
12 2020-05-19  Commemoration of Atatürk, Youth and Sports Day
13 2020-07-15                Democracy and National Unity Day
14 2020-08-30                                     Victory Day
15 2020-10-29                                    Republic Day
16 2020-05-23                                     Eid al-Fitr
```

I think the problem is caused here:
https://github.com/facebook/prophet/blob/master/python/fbprophet/hdays.py#L971
and I guess the issue might be resolved by applying the `offset` on `islam_year` rather than `year`.

Compare the following code snippets and corresponding outputs to see the difference:

Offset `year`:
```python
for offset in range(-1, 2, 1):
    islam_year = from_gregorian(year + offset, 8, 22)[0]
    y, m, d = to_gregorian(islam_year, 12, 10)
    print(y, m, d)
```
```
2019 8 12
2021 7 20
2022 7 10
```
Offset `islam_year`:
```python
islam_year = from_gregorian(year, 8, 22)[0]
    for offset in range(-1, 2, 1):
        y, m, d = to_gregorian(islam_year + offset, 12, 10)
        print(y, m, d)
```
```
2020 7 31
2021 7 20
2022 7 10
```

Thanks!

**Comment by bletham** [comment]
Thanks for digging into this and figuring out what's happening. Yes putting the offset on `islam_year` does seem to be the right fix.

On a related note - the `holidays` package seems to have added Turkey, but have it under `TR` instead of the `TU` that we used. So if you do this with `TR` you will get the holidays from there, and it does include 'Sacrifice Feast Holiday' for 2020. It actually includes 3 days for the holiday. Since we use the `holidays` package anyway, I think now that it has Turkey it'd be good to use those instead of duplicating, and perhaps we can make `TU` just an alias for `TR`.

**Comment by bletham** [comment]
This is fixed in 6ec0a90 by using the holidays package as described.

**Comment by bletham** [comment]
Fix has been pushed to pypi in the latest v1.0 release
